meson.build

```
@@ -54,3 +57,95 @@ pgroonga_check = shared_module('pgroonga_check',
   pgroonga_check_sources,
   kwargs: pgrn_module_args,
 )
+
+pgroonga_database_sources = files(
+  'src/pgroonga-database.c',
+)
+
+pgroonga_database = shared_module('pgroonga_database',
+  pgroonga_database_sources,
+  kwargs: pgrn_module_args,
+)
+
+install_data('pgroonga_database.control',
+  install_dir: pg_sharedir / 'extension',
+)
+configure_file(
+  input: 'data/pgroonga_database.sql',
+  output: f'pgroonga_database--@pgrn_version@.sql',
+  copy: true,
+  install: true,
+  install_dir: pg_sharedir / 'extension',
+)
+
+pgroonga_database_upgrade_downgrade_sqls = files(
```

Can we simplify this?

```suggestion
pgroonga_database_update_sqls = files(
```